api.md

```diff
@@ -1906,9 +1949,20 @@ The `start_offset` and `end_offset` parameters should be specified differently d
 |`job_id`| (INTEGER)  TimescaleDB background job id created to implement this policy|
 
 
+#### Errors
+
+An error will be given if:
+
+- The `start_offset` or `end_offset` specify a range less than two buckets in size.
+- The type of `start_offset` or `end_offset` is not compatible with
+  the time type of the continuous aggregate.
+
 #### Sample Usage [](add_continuous_aggregate_policy-examples)
 
-Add a policy that refreshes the last month once an hour, excluding the latest hour from the aggregate (for performance reasons, it is recommended to exclude buckets that still see lots of writes):
+Add a policy that refreshes the last month once an hour, excluding the
+latest hour from the aggregate (for performance reasons, it is
+recommended to exclude buckets that still see lots of writes):
```

Comment:
IMHO, it flows better if the comment in parentheses is after the sample code and without parentheses.

Reply:
Still suggest to move the comment in the parentheses to after the code sample.